insights/combiners/tests/test_httpd_conf.py

```diff
@@ -153,21 +217,8 @@ def test_httpd_one_file_overwrites():
     shared = {HttpdConf: [httpd]}
     result = HttpdConfAll(None, shared)
 
-    assert result.get_valid_setting('UserDir') == ('enable bob', 'httpd.conf')
-
     active_setting = result.get_active_setting('UserDir')
     assert active_setting.value == 'enable bob'
     assert active_setting.line == 'UserDir enable bob'
     assert active_setting.file_path == '/etc/httpd/conf/httpd.conf'
     assert active_setting.file_name == 'httpd.conf'
-
```

Comment:
Please do not remove method tests for valid methods. This is expected behavior. See https://bugzilla.redhat.com/show_bug.cgi?id=1455516 .